tests/analyses/new/test_systemviewers.py

```diff
@@ -12,6 +12,14 @@
 from zospy.analyses.new.systemviewers.base import SystemViewerWrapper
 
 
+def consider_minimal_opticstudio_version_in_result(result, minimal_version):
```

Comment:
I'd choose a name that shows that the assert happens in this function, e.g. `assert_systemviewer_result`

Reply:
Name is changed to a more concise one!